lib/migrator.js

```diff
@@ -81,43 +106,36 @@ internals.symlinkMigrationFile = async (migrationFile, symlinkConfig) => {
         return;
     }
 
-    const file = Path.join(symlinkConfig.migrationsDir, migrationFile);
+    const file = Path.join(symlinkConfig.migrationDir, migrationFile);
     const link = Path.join(symlinkConfig.tmpPath, migrationFile);
 
-    await internals.PROMISIFIED_UTILITIES.symlinkAsync(file, link);
-};
-
-exports.migrate = async (knexGroups, defaultKnex, rollback) => {
-
-    const knexMigrations = new Map();
-
-    // Compile migration info per knex instance
-
-    for (let i = 0; i < knexGroups.length; ++i) {
-        const knexGroup = knexGroups[i];
-        const migrationsDir = knexGroup.migrationsDir;  // Already normalized to abs path
+    await Util.promisify(Fs.symlink)(file, link);
 
-        if (!migrationsDir) {
-            continue;
-        }
+};
 
-        const knex = knexGroup.knex || defaultKnex;
+/**
+ * A non-standard one :) Wraps the Tmp.dir method in a promise. We do this manually instead of using Util.promisify as we do with the other callback-style methods
+ * because Tmp.dir's callback takes 2 non-error parameters, but Promises fulfill with only a single value and Util.promisify handles this by
+ * fulfilling with only the first parameter (the path to the temporary directory), which resulted in us being unable to cleanup our temporary directory because we no longer had access to Tmp's cleanup method
+ *
+ * @param {object} opts — Options object that Tmp.dir normally expects
```

Comment:
The comment is fine, but can we get rid of the doc block?  I find it's all or nothing with those suckers.